requirements/pytorch/test.txt

```diff
@@ -1,7 +1,7 @@
 coverage ==7.3.0
 pytest ==7.4.0
 pytest-cov ==4.1.0
-pytest-forked ==1.4.0
+pytest-forked ==1.6.0
```

Comment:
@Borda I don't think we use this dependency anymore. It was added in https://github.com/Lightning-AI/lightning/pull/11808

```suggestion
```